docs/news/.gitignore

```
@@ -0,0 +1,25 @@
+.Rproj.user
+.Rhistory
+.RData
+.DS_Store
+kindlegen*
+_book
+rsconnect
+CMake*
+!CMakeLists.txt
+Makefile
+_bookdown.yml
+cover.tex
+build-book.sh
+cmake_install.cmake
+rstudio-server-pro-*-admin-guide.*
+_main.html
+*.zip
+
+index_files/**
+_variables.yml
+_site/
+_publish.yml
+/.quarto/
+.bash_history
+.cache/*
```

I don't think we need a lot of these. I realize you copied this over from a different doc folder, where they're probably also unnecessary, but best not to create more clutter. If this is the only thing holding up merging this PR then I can take care of it in my build PR. 
```suggestion
.DS_Store
kindlegen*
rsconnect
CMake*
!CMakeLists.txt
Makefile
cmake_install.cmake

_site/
_publish.yml
/.quarto/
.bash_history
.cache/*
```